docs/shared/packages/gradle/gradle-plugin.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Overview of the Nx Gradle Plugin
+description: The Nx Gradle plugin contains generators and plugins that are useful for Gradle projects in an Nx workspace.
+---
+
+The Nx Gradle plugin contains generators and plugins that are useful for Gradle projects in an Nx workspace.
```

Comment:
I also think we need a section on why one would use Nx and Gradle together.

It should include information such as:
* Being able to use the Nx cache on top of the Gradle cache to speed up tasks
* Being able to distribute gradle tasks via Nx Cloud
* Being able to view the project graph via Nx including relationships to non-Gradle projects